app/src/main/res/layout/activity_numeric_input_interaction_view_test.xml

```diff
@@ -21,4 +21,31 @@
     android:maxLength="200"
     android:padding="8dp"
     android:singleLine="true" />
+  <org.oppia.app.customview.interaction.TextInputInteractionView
+    android:id="@+id/test_text_input_interaction_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp"
+    android:background="@drawable/edit_text_background"
+    android:focusable="true"
+    android:hint="Write here."
+    android:inputType="text"
+    android:longClickable="false"
+    android:maxLength="200"
+    android:padding="8dp"
+    android:singleLine="true" />
+  <org.oppia.app.customview.interaction.FractionInputInteractionView
+    android:id="@+id/test_fraction_input_interaction_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp"
+    android:background="@drawable/edit_text_background"
+    android:focusable="true"
+    android:digits="0123456789/-"
```

Comment:
I wonder if we should performing additional validation. Perhaps that's something we can address as part of #33.

Reply:
Ya we need additional validations beside this hygiene validation. I would like to know what all validations i need to take care beside the UI hygiene validation.

Reply:
created separate string to Fraction Parser and further validation would be implemented in upcoming PR